server/datastore/mysql/policies.go

```diff
@@ -292,16 +292,7 @@ func listPoliciesDB(ctx context.Context, q sqlx.QueryerContext, teamID, countsFo
 		ctx,
 		q,
 		&policies,
-		fmt.Sprintf(`SELECT p.id,
-      p.team_id,
-      p.resolution,
-      p.name,
-      p.query,
-      p.description,
-      p.author_id,
-      p.platforms,
-      p.created_at,
-      p.updated_at,
+		fmt.Sprintf(`SELECT p.*,
```

Comment:
We discussed not doing `SELECT *` style selects: https://fleetdm.slack.com/archives/C019WG4GH0A/p1663615428272069

Reply:
Thank you for bringing this up. I didn't know about that discussion. I'll then list the columns instead.

I made this consistent with line 62 that uses it and just worked, after I at first forgot to update one of these queries here and only found the problem after manual testing plus work to find the actual problem. I personally have my doubts about the fitness of humans (including me) to maintain repeated list of columns consistently.

Reply:
Agreed, neither is a bulletproof system.